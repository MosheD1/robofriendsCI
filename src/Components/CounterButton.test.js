import {shallow, mount, render} from 'enzyme';
import React from 'react';
import CounterButton from './CounterButton';

it('expect state to have a count', () => {
    const wrapper = shallow(<CounterButton />);
    expect(wrapper.state('count')).toBe(0);
});

it('expect shouldComponentUpdate to run', () => {
    const wrapper = shallow(<CounterButton />);
    const instance = wrapper.instance();
    instance.updateCount();
    expect(instance.state.count).toBe(1);
})

it('expect to render CounterButton', () => {
    expect(shallow(<CounterButton />)).toMatchSnapshot();
});