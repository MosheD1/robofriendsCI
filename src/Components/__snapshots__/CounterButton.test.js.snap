// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CounterButton 1`] = `ShallowWrapper {}`;
